downstream/modules/platform/ref-controller-database-settings.adoc

```diff
@@ -0,0 +1,67 @@
+[id="ref-controller-database-settings"]
+
+= Database Settings
+
+The following are configurable settings in the database that can help to improve performance:
```

Comment:
Suggest rewording to:
Configure the following settings in the database to improve performance: